### DISCUSSION TITLE
How to use React-Time-Picker with Next.js?

### DISCUSSION THREAD

**Discussion by neaorin** [wojtekmaj/react-time-picker#134] [Q&A] [answered]
Trying to use TimePicker in a [Next.js](https://nextjs.org/) project, I get the following error:

**Failed to compile**

../../node_modules/react-clock/dist/Clock.css
Global CSS cannot be imported from within node_modules.
Read more: https://nextjs.org/docs/messages/css-npm
Location: ..\..\node_modules\react-time-picker\dist\entry.js

Is it possible to ship a compiled version of Clock.css?

Thank you.

**Comment by wojtekmaj** [comment, marked as answer]
In Next.js, you'd need to import CSS yourself by following these instructions:

https://github.com/wojtekmaj/react-time-picker#custom-styling

Or don't import CSS at all and create your own styles as you like.

**Reply by subr1oto** [reply]
How to make clock dynamic as implemented in https://mui.com/x/react-date-pickers/time-picker/

**Reply by wojtekmaj** [reply]
> How to make clock dynamic as implemented in https://mui.com/x/react-date-pickers/time-picker/

You can't, sorry
